Dockerfile.net-utils

```diff
@@ -0,0 +1,23 @@
+FROM registry.fedoraproject.org/fedora:latest
```

Comment:
Since we have a requirement to pull the base image from ACR, we could potentially use one of our existing images here, and instead add a fedora repository to it in your dockerfile to install the pagkages needed.

This is currently being done in https://github.com/Azure/ARO-RP/pull/2567 to install openvpn since it was not available in the base image.

Reply:
In that PR we're installing openvpn from EPEL rather than Fedora proper. One approach would be to maintain EPEL packages for the tools we want to have available.

Reply:
The only compliance requirement that would hit us on this change that I am aware of is the FROM directive pointing to something other than our ACRs - adding repositories or pulling down packages later in the dockerfile should be fair game. Do you mean that we could host the packages somewhere on our end?